### DISCUSSION TITLE
Change the Dockerfile to build the latest nominatim's master

### DISCUSSION THREAD

**Discussion by roeezamir** [mediagis/nominatim-docker#331] [Q&A]
Hey!

(Don't know if i should ask it here or in the nominatim's repo)
I've tried to change the Dockerfile (of 4.0)  to build the latest nominatim's master by adding these lines:
```
    && git clone --recursive https://github.com/openstreetmap/Nominatim.git \
    && wget -O Nominatim/data/country_osm_grid.sql.gz https://www.nominatim.org/data/country_grid.sql.gz \
```
And little more tiny required changes so it will work.
(Instead of downloading and extracting the tar.bz2 file from  [](https://nominatim.org/release))

The build was finished successfully but when I tried to run the image, 
it failed at a very late stage:
```
Checking indexing status ... Failed
The indexing didn't finish. 33 entries are not yet indexed.

To index the remaining entries, run:   nominatim index

Checking that database indexes are complete ... OK
Checking that all database indexes are valid ... OK
Checking TIGER external data table. ... OK
```
And the docker run fails
Any idea how can i make it work?

Thanks :)

**Comment by leonardehrenfried** [comment]
There was someone who opened a PR for doing what you are trying: https://github.com/mediagis/nominatim-docker/pull/290

We didn't want to merge it into our repo and advised him to maintain it in his own account. It's worth asking him if he ended up doing it.

**Reply by roeezamir** [reply]
Great exactly what I was needed!
But eventually, looks like my Dockefile is ok and the error is due to some issue with the latest nominatim commit.

Thanks for the help!

**Comment by mtmail** [comment]
> The indexing didn't finish. 33 entries are not yet indexed.

In rare cases some places remain unindexed after an import. Running `nominatim index` again usually fixes that, it will only do those 33 and be very fast. It happens when indexing a place which then marks other dependend places as to-be-indexed.

**Reply by roeezamir** [reply]
The problem is that the ducker run fails (i assumed that because of this error).
Anyway, it happens in the latest nominatim's master commit. 
But, if I checkout to a previous commit (8a0e3e2 for e.g.) it works perfectly fine.

Thanks for the help!
